File: algorithm/src/main/java/baekjoon/p05666/Main.java
package baekjoon.p05666;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while (sc.hasNext()) {
            System.out.println(solve(sc.nextInt(), sc.nextInt()));
        }
    }

    private static String solve(int h, int p) {
        return String.format("%.2f", (double) h / p);
    }

}

File: algorithm/src/main/java/baekjoon/p05666/Main.kt
package baekjoon.p05666

fun main(args: Array<String>) {
    while (true) {
        val input = readLine() ?: break
        val inputs = input.split(" ").map { it.toInt() }
        println(solve(inputs[0], inputs[1]))
    }
}

private fun solve(h: Int, p: Int): String {
    return "%.2f".format(h.toDouble() / p)
}